src/components/Routes.js

```diff
@@ -31,6 +33,8 @@ const Routes = () => (
     <Route exact path="/google/social-login" component={verifyAuth} />
     <Route exact path="/twitter/social-login" component={verifyAuth} />
     <Route exact path="/facebook/social-login" component={verifyAuth} />
+    <Route exact path="/reset" component={ResetRequest} />
+    <Route exact path="/response" component={Feedback} />
```

Comment:
It would be good if you use a descriptive path

Reply:
@SokoPaulSokool ,
Paths renamed to resetpassword and resetpasswordresponse respectively.
A small modification though will be needed in the backend link sent in email.
Will raise it with @salviosage

Reply:
@SokoPaulSokool, a simple growing-ribbon progress bar added to the top of the page, growing from    0-length, red color to full-length green every 2 seconds while still processing data

Reply:
Also added mobile view pictures